tools/cli/adaparse.cpp

```diff
@@ -140,6 +136,15 @@ int piped_file(Callable&& adaparse_print, const cxxopts::ParseResult result,
   }
   if (offset > 0) {
     // have a line of length offset at cachebuffer.get()
+    std::string_view line(cachebuffer.get(), offset);
+
+    ada::result<ada::url_aggregator> url = ada::parse(line);
+    if (!url) {
+      adaparse_print("Invalid URL:{}\n", line);
+    } else if (!get_part.empty()) {
+      print_part(adaparse_print, get_part, url.value());
```

Comment:
what is the difference between `adaparse_print` and `print_part`? I think this line should be `adaparse_print_part` to be consistent with the previous function name?